apps/hexagon_matmul/Makefile

```diff
@@ -0,0 +1,55 @@
+include ../support/Makefile.inc
```

Comment:
If you put all of the intermediate stuff into a BIN subdir (a la apps/simd_op_check), .gitignore won't need updating and the world will be a happy place.